_episodes/15-carpentries.md

```diff
@@ -0,0 +1,297 @@
+---
```

Comment:
Note: the diff for How We Operate is not viewable here due to episode renumbering. Most changes can be viewed in the [earlier PR to the sandbox repo here.](https://github.com/data-lessons/instructor-training/pull/39/files)